sqlx/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sqlx.magic import load_ipython_extension
```

Comment:
I'm curious. Since this directory is not in the singlestoredb directory, is it getting installed as its own module in site-packages? I guess it's fine to install it as a regular package since it's just doing what the `%sql` magic is doing it in a multi-threaded way. I just wanted to verify that this was the intended behavior.

Reply:
Yes, this is so that user can just run `%load_ext sqlx` instead of `%load_ext singlestoredb.sqlx` in Jupyter Notebook to load the extension. This is same behavior as jupysql which allows loading extension via `%load_ext sql`.